test/schema/export_test.js

```diff
@@ -21,14 +24,27 @@ describe('#exportToSchemasUniqueCases()', () => {
     let schemas = exportToSchemas([ns]);
     expect(schemas).to.have.length(16);
     expect(schemas).to.eql(expectedSchemas);
+    expect(validateSchema(schemas));
```

Comment:
I tested this by catching a multi-element array mid-execution, and the validate function does indeed make sure each element in the array conforms. I can explicitly make it iterate, but watching the execution seemed to make that redundant.